pytradfri/resource.py

```diff
@@ -53,7 +53,7 @@ class DeviceInfoResponse(BaseModel):
     battery_level: int | None = Field(alias=ATTR_DEVICE_BATTERY)
 
 
-class ApiResourceResponse(BaseModel):
+class ResourceResponse(BaseModel):
```

Comment:
We want a base ApiResourceResponse that doesn't have any device specific attributes. Then make a separate response model for the Device class that has the additional device attributes.

Reply:
Example:

```py
class ApiResourceResponse(BaseModel):
    """Represent a resource response."""

    id: int = Field(alias=ATTR_ID)
    name: str = Field(alias=ATTR_NAME)
    created_at: Optional[int] = Field(alias=ATTR_CREATED_AT)
```

Reply:
Thank you!